material-overrides/main.html

```diff
@@ -195,63 +195,103 @@
     });
   </script>
   <script>
+    // AI_RAW_BASE = base URL where AI artifacts are stored (injected via mkdocs.yml:extra)
+    const AI_RAW_BASE =
+      "{{ config.extra.ai_raw_base | default('https://raw.githubusercontent.com/polkadot-developers/polkadot-docs/master/.ai') }}";
```

Comment:
```suggestion
     'https://raw.githubusercontent.com/polkadot-developers/polkadot-docs/master/.ai';
```

So we removed `ai_raw_base` from the mkdocs.yml config file, so this will always fall back to `https://raw.githubusercontent.com/polkadot-developers/polkadot-docs/master/.ai` so let's just update it to be what it is